src/app/api/onramp-stripe/route.tsx

```diff
@@ -9,7 +9,7 @@ export async function GET(req: Request) {
         const response = await fetch('https://api.stripe.com/v1/crypto/onramp_sessions', {
             method: 'POST',
             headers: {
-                'Authorization': `Bearer ${process.env.STRIPE_SECRET_KEY_TEST}`,
+                'Authorization': `Bearer ${process.env.STRIPE_SECRET_KEY}`,
```

Comment:
How do we handle this in development? will it fail if they key is not present?

I notice there is still a STRIPE_SECRET_KEY_TEST, is it needed? if not, we should remove it, we should not keep deprecated keys that are not used, for several reasons.

Reply:
I have serious doubts about this solution. It seems we are just skipping the dev environment and switching everything to production, without distinction, is this the best approach we can do?